src/shared/components/ResourcesList/ResourcesList.js

```diff
@@ -116,14 +116,7 @@ export function ResourcesList({
           />
         }
       />
-      {resources ? (
-        <ResourceListRenderer
-          resources={(resources || []).filter(filterFn)}
-          {...allProps}
-        />
-      ) : (
-        <Resources {...allProps} />
-      )}
```

Comment:
This couldn't be removed when you create an extension with some filter function, so it won't work. Now you are just taking this prop, pass, and don't make anything with it.